service/src/test/resources/application-test.yml

```diff
@@ -1,6 +1,6 @@
 workspace:
   crl:
-    janitor-client-credential-file-path: rendered/janitor-client-sa-account.json
+    janitor-client-credential-file-path: ../config/janitor-client-sa.json
```

Comment:
I still think `config` is where I would look for all kinds of parameters and constants. This directory is more of a `credentials` or something.